spyder/plugins/variableexplorer/widgets/collectionseditor.py

```diff
@@ -1817,6 +1825,6 @@ def remote_editor_test():
     app.exec_()
 
 
-if __name__ == "__main__":
-    editor_test()
-    remote_editor_test()
+# if __name__ == "__main__":

```

Comment:
Should this be commented out? If needed, could you explain why please?

Reply:
aah, I forgot to comment it back in. I can't run the file (and put the declarations in the workspace)